### ISSUE TITLE
Check failed: input.part_size() == 1 (0 vs. 1)

### ISSUE THREAD

**Issue by wormsleep** [dsindex/syntaxnet#13]
I have modified CDIR and PID as absolute paths in Mac OS X, but when run parser_trainer_test.sh there is an error arises below

(tensorflow) zhaoweideMacBook-Pro:work zhaowei$ ./parser_trainer_test.sh 
- CDIR=/Users/zhaowei/tensorflow/models/syntaxnet/work
- PDIR=/Users/zhaowei/tensorflow/models/syntaxnet
- cd /Users/zhaowei/tensorflow/models/syntaxnet
- SYNTAXNET_HOME=/Users/zhaowei/tensorflow/models/syntaxnet
- BINDIR=/Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet
- CONTEXT=/Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/context.pbtxt
- TMP_DIR=/Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output
- mkdir -p /Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output
- cat /Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/context.pbtxt
- sed s=OUTPATH=/Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output=
- PARAMS=128-0.08-3600-0.9-0
- /Users/zhaowei/tensorflow/models/syntaxnet/bazel-bin/syntaxnet/parser_trainer --arg_prefix=brain_parser --batch_size=32 --compute_lexicon --decay_steps=3600 --graph_builder=greedy --hidden_layer_sizes=128 --learning_rate=0.08 --momentum=0.9 --output_path=/Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output --task_context=/Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output/context --training_corpus=training-corpus --tuning_corpus=tuning-corpus --params=128-0.08-3600-0.9-0 --num_epochs=12 --report_every=100 --checkpoint_every=1000 --logtostderr
  INFO:tensorflow:Computing lexicon...
  I syntaxnet/lexicon_builder.cc:134] Term maps collected over 964 tokens from 54 documents
  I syntaxnet/term_frequency_map.cc:139] Saved 547 terms to /Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output/word-map.
  I syntaxnet/term_frequency_map.cc:139] Saved 529 terms to /Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output/lcword-map.
  I syntaxnet/term_frequency_map.cc:139] Saved 36 terms to /Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output/tag-map.
  I syntaxnet/term_frequency_map.cc:139] Saved 36 terms to /Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output/category-map.
  I syntaxnet/term_frequency_map.cc:139] Saved 39 terms to /Users/zhaowei/tensorflow/models/syntaxnet/work/testdata/tmp/syntaxnet-output/label-map.
  F syntaxnet/task_context.cc:140] Check failed: input.part_size() == 1 (0 vs. 1)char-map
  ./parser_trainer_test.sh: line 38:  3247 Abort trap: 6           "$BINDIR/parser_trainer" --arg_prefix=brain_parser --batch_size=32 --compute_lexicon --decay_steps=3600 --graph_builder=greedy --hidden_layer_sizes=128 --learning_rate=0.08 --momentum=0.9 --output_path=$TMP_DIR --task_context=$TMP_DIR/context --training_corpus=training-corpus --tuning_corpus=tuning-corpus --params=$PARAMS --num_epochs=12 --report_every=100 --checkpoint_every=1000 --logtostderr

**Comment by xtknight** [comment]
This issue happens because "char-map" is not specified in the context.pbtxt_p file or the path does not exist. Try adding the following to the context.pbtxt_p file. Or just copy another entry like word-map and change to char-map (all the entries should be similar to each other).

    input {
      name: "char-map"
      Part {
        file_pattern: "OUTPATH/char-map"
      }
    }
